### DISCUSSION TITLE
Angle 90º 180º 270º

### DISCUSSION THREAD

**Discussion by newleader77** [roleoroleo/yi-hack-Allwinner-v2#112] [Ideas]
Hi there.
There is a function to work with camera if place on top and invert image. Is it possible to do the same but in different angles.
If yes, is it possible to work with 90º and 270º.

Best Regards.

**Comment by roleoroleo** [comment]
Sorry, is not possible.

**Comment by newleader77** [comment]
Ok Tanks for fast reply.
